integration_test/third_party_apps_data/applications/hbase/sles/post

```diff
@@ -0,0 +1,3 @@
+set -e
+
+sudo systemctl restart hbase 
```

Comment:
Looks like the `submodules/fluent-bit` change below got included by accident. Can we remove it from the change set?

Reply:
Removed